crates/oxc_linter/src/rules/import/namespace.rs

```diff
@@ -24,7 +26,41 @@ declare_oxc_lint!(
 );
 
 impl Rule for Namespace {
-    fn run_once(&self, _ctx: &LintContext<'_>) {}
+    fn run(&self, node: &AstNode<'_>, ctx: &LintContext<'_>) {
```

Comment:
Feels like we can test on each `MemberExpression` and see if the lhs identifier reference has the `import` flag?

Reply:
No, Because It's not enough to have an `import` flag, we need to check if it's an `ImportNamespaceSpecifier`.